org.eclipse.egit.core/src/org/eclipse/egit/core/Activator.java

```diff
@@ -248,10 +248,20 @@ public void optionsChanged(DebugOptions options) {
 	 * @since 4.1
 	 */
 	public MergeStrategy getPreferredMergeStrategy() {
+		// Get preferences set by user in the UI
 		final IEclipsePreferences prefs = InstanceScope.INSTANCE
 				.getNode(Activator.getPluginId());
 		String preferredMergeStrategyKey = prefs.get(
 				GitCorePreferences.core_preferredMergeStrategy, null);
+
+		// Get preferences defined at eclipse startup with configuration files
+		if (preferredMergeStrategyKey == null
```

Comment:
That's very interesting... We probably need to do the same in EMFCompare to allow out users to define default properties wherever they want, don't we?